token-lending/program/src/processor.rs

```diff
@@ -982,6 +983,18 @@ fn process_refresh_obligation(program_id: &Pubkey, accounts: &[AccountInfo]) ->
         }
 
         liquidity.accrue_interest(borrow_reserve.liquidity.cumulative_borrow_rate_wads)?;
+        max_borrow_weight = match max_borrow_weight {
+            None => Some((borrow_reserve.borrow_weight(), index)),
+            Some((cur_max_weight, _)) => {
+                if borrow_reserve.borrow_weight() > cur_max_weight
```

Comment:
if this was >=, the ordering of the borrows could change after every refresh_obligation call, which would be pretty bad.